helita/sim/bifrost.py

```diff
@@ -745,6 +1758,8 @@ def write_multi3d(self, outfile, mesh='mesh.dat', desc=None,
             z.tofile(fout2, sep="  ", format="%11.5e")
             fout2.close()
 
+
+class create_new_br_files():
```

Comment:
Please follow the convention of Uppercase for class names. And remove `()` from class statement if not inheriting.